powa_collector/snapshot.py

```diff
@@ -138,12 +152,18 @@ def copy_remote_data_to_repo(cls, data_name,
     if (cls.is_stopping() or not src_ok):
         return errors
 
-    # insert the data to the transient unlogged table
+    # insert the data to the local temp table
+    temp_table_name = get_table_basename(target_tbl_name, data_ins) + '_' + str(srvid)
```

Comment:
You should modify `get_global_tmp_name()` to return a tuple with the schema name and the table name separately so there is no need to try to recover the table name afterwards.